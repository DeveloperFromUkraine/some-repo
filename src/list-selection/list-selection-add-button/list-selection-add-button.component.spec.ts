import { ComponentFixture, TestBed } from "@angular/core/testing";
import { ListSelectionAddButtonComponent } from "index";
import { ComponentTest } from "../../../test/test-bed/component";

describe('ListSelectionAddButtonComponent', () => {
    let fixture: ComponentFixture<ListSelectionAddButtonComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [ListSelectionAddButtonComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(ListSelectionAddButtonComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});