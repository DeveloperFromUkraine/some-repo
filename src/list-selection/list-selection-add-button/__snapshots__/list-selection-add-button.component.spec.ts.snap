// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Selection Add Button should match snapshot 1`] = `
<ign-list-selection-add-button>
  <a
    mat-fab=""
  >
    
    
    <mat-icon
      aria-hidden="true"
      class="mat-icon"
      role="img"
    >
      
      add
    
    </mat-icon>
    
  
  </a>
  
</ign-list-selection-add-button>
`;
